chat/client-libraries/cloud/src/main/java/com/google/workspace/api/chat/samples/CreateMessageAppCred.java

```diff
@@ -32,10 +45,58 @@ public static void main(String[] args) throws Exception {
         CreateMessageRequest.newBuilder()
         // Replace SPACE_NAME here.
         .setParent("spaces/SPACE_NAME")
-        .setMessage(
-            Message.newBuilder()
-            .setText("Hello with app credentials!")
+        .setMessage(Message.newBuilder()
+          .setText( "👋🌎 Hello world! I created this message by calling " +
+                    "the Chat API\'s `messages.create()` method.")
+          .addCardsV2(CardWithId.newBuilder().setCard(Card.newBuilder()
```

Comment:
For these Card/Section/Widget builders, I think the add/set methods can accept a builder as parameter. So, we can clean it up a bit by removing all the intermediate `build()`, and leaving only the one from the `Message` builder.

Reply:
Good call. Will apply this improvement to all other samples in a separate PR.